core/internal/config/dms.go

```diff
@@ -31,7 +31,7 @@ func LocateDMSConfig() (string, error) {
 
 	for _, dir := range strings.Split(configDirs, ":") {
 		if dir != "" {
-			primaryPaths = append(primaryPaths, filepath.Join(dir, "quickshell", "dms"))
+			primaryPaths = append(primaryPaths, filepath.Join(dir, "quickshell", "dms", "quickshell"))
```

Comment:
Does it not account for it? It keeps both paths in the search interleaved, so it checks non mono repo path, then monorepo path in the same order as the higher level directories.

Reply:
You need to build/download v0.5.1 then it should be good. That was the first monorepo release